### ISSUE TITLE
HA-Bridge to bridge plugs to harmony ?

### ISSUE THREAD

**Issue by RY00** [bwssytems/ha-bridge#956]
Hi,

First of all, thanks to all contributors of the project, I read so many things from so many peoples while trying setup my ha-bridge and it's still a pleasure to see a such community !
Second I apologie if it's not to good place for my question, otherwise do not hesitate to tag it as a question (for newbie I mean).
I'm trying to set up ha-bridge on a Windows computer to bridge some plugs (sonoff S20 with tasmota).
My goal is to manage my plugs from my harmony hub with remote/activities, but also from my google home (not sure it's possible, but I will try using the harmony hub as a brige telling to google to perform action on harmony).
I free up the port 80, set up a device through the web interface and tested On/Off commands and it works.
My problem is that when I try to add the ha-bridge to my harmony, it does not discover ha-bridge.
The only log I have is:
Traceupnp: sendUpnpNotify notifyTemplate is <<<NOTIFY * HTTP/1.1
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 LOCATION: http://198.9.1.1:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
 NTS: ssdp:alive
 hue-bridgeid: 60A44CFFFE3D46D3
 NT: uuid:2f402f80-da50-11e1-9b23-60a44c3d46d3
 USN: uuid:2f402f80-da50-11e1-9b23-60a44c3d46d3
 >>>
I have try to disable firewall, allow UPnP on my router, add the link button and on several ha-bridge version but nothing worked.
I have no idea on how to investigate this, does someone has an idea ?
Regards,

**Comment by audiofreak9** [comment]
Did you click the link button when you tried to have the Harmony discover your ha-bridge?  

Also from the Bridge Control tab, what Emulate Hue Hub Version is displayed?  The current version i s 1802201122.

**Comment by RY00** [comment]
I tested what you suggest but finally I found another source to the problem.
I got another UPnP application (Universal Media Server) running on the same computer, I switched it OFF and my harmony found ha-bridge them pair with it.
Afterward I switched back ON Universal Media Server and both application continue to work fine.
Finally I set ha-bridge as a Windows service by using winsw and all works fine !
Thanks for the help.
Regards,
